protocols/index.js

```diff
@@ -67,7 +68,7 @@ import soldat from './soldat.js'
 export {
   armagetron, ase, asa, assettocorsa, battlefield, buildandshoot, cs2d, discord, doom3, eco, epic, factorio, farmingsimulator, ffow,
   fivem, gamespy1, gamespy2, gamespy3, geneshift, goldsrc, gtasao, hexen2, jc2mp, kspdmp, mafia2mp, mafia2online, minecraft,
-  minecraftbedrock, minecraftvanilla, minetest, mumble, mumbleping, nadeo, openttd, palworld, quake1, quake2, quake3, rfactor, ragemp, samp,
+  minecraftbedrock, minecraftvanilla, minetest, mumble, mumbleping, nadeo, openttd, palworld, quake1, quake2, quake3, renegadex, renegadexmaster, renegadexmaster as renegade10master, rfactor, ragemp, samp,
```

Comment:
Export only one entry, which should be named as `${game_id}master`, so `renegadexmaster`.
(side note: this is also not specified anywhere nor standalized unfortunately)

Reply:
That's the issue as the _game id_ is forced to be `...10`. `renegadex...` would be wrong to due to  the strict _normalisation_. Exporting the "normalized" name was just a shortcut solution.